testdata/project-v3-addon/Makefile

```diff
@@ -2,7 +2,7 @@
 # Image URL to use all building/pushing image targets
 IMG ?= controller:latest
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
-CRD_OPTIONS ?= "crd:trivialVersions=true"
+CRD_OPTIONS ?= "crd:trivialVersions=true,preserveUnknownFields=false"
```

Comment:
Shows that `preserveUnknownFields=false` is only required when the crd version is not v1. Also,should not it be scaffolded using v1 as default?

Reply:
https://github.com/kubernetes-sigs/kubebuilder/pull/1644#discussion_r474282404